vdm-io/Joomla-Component-Builder
# Moved from PHPExcel library to PHPspreadsheet

Pull Request for Issue gh-541, gh-529, gh-540, gh-518, gh-444.

### Summary of Changes
- Fixed gh-541 that the target client value for admin area is correctly set as administrator.
- Fixed gh-529 so that the custom admin view does not hidemainmenu when it is selected as the default view (dashboard)
- Fixed gh-540 to allow for html in the translation field.
- Resolved gh-518 to allow DECIMAL values as the Other Data Length value.
- Resolved gh-444 by adding the new phpspreadsheet library, and removing the old PHPExcel library.

### Testing Instructions
- See each issue for instructions.
- Specifically test the new PHPspreadsheet library that should work as part of the export and import function in each admin view with that default features.

### Expected result
- That all issues are resolved or fixed.
- That with the new PHPspreadsheet library the import and export of data from the database in the related admin view work as expected.

### Actual result
- See each issue
- We were in an old version of PHPExcel and now upgraded to PHPspreadsheet

### Documentation Changes Required
None needed
